src/routes/ViewRoute.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
```

Comment:
I don't understand what `<ViewRoute>` is for.

Reply:
It's the wrapper with the header and the switch between flow and details.